### DISCUSSION TITLE
Disabling NAT for IP allocation, but not for eNB communication?

### DISCUSSION THREAD

**Discussion by jpad-freedomfi** [magma/magma#7965] [Q&A]
Hi, 

We have some customers who have eNB plugged into AGW eth1, want IP allocation to be DHCP from upstream router (north of eth0/SGi interface), but still want NAT enabled so eNB can reach internet. 

Is this possible? The use case is CBRS SAS connection for eNB connected on eth1 interface.

Is it possible to decouple IP address allocation method and NAT enable/disable? 

Thanks,
Joey

**Comment by jpad-freedomfi** [comment]
@pshelar Amar said this would be new feature ask. Any comments or thoughts on this? Otherwise I can make a proposal.

**Reply by amarpad** [reply]
cc @pshelar this is the management L2 that you were talking about, then in there we can split the DHCP allocation user plane from the eNB traffic.

**Reply by pshelar** [reply]
> cc @pshelar this is the management L2 that you were talking about, then in there we can split the DHCP allocation user plane from the eNB traffic.

sure, we can allocate DHCP address for eNB. that should not be hard.

**Comment by pshelar** [comment]
> Hi,
> 
> We have some customers who have eNB plugged into AGW eth1, want IP allocation to be DHCP from upstream router (north of eth0/SGi interface), but still want NAT enabled so eNB can reach internet.
> 
So in this case, eNB gets IP address from DHCP server in SGi network. if that is case, why not allow router NAT take care of eNB internet traffic?

> Is this possible? The use case is CBRS SAS connection for eNB connected on eth1 interface.
> 
> Is it possible to decouple IP address allocation method and NAT enable/disable?
> 
> Thanks,
> Joey

**Reply by jpad-freedomfi** [reply]
Wait, that's not the case. If we plug eNB into eth1 of AGW eNB gets IP from dnsmasq service in AGW. With NAT disabled it can't ping passed AGW.

**Reply by pshelar** [reply]
@jpad-freedomfi right.

Can you try this PR: https://github.com/magma/magma/pull/8134

**Reply by pshelar** [reply]
@amarpad @jpad-freedomfi can you try the PR for 1.5 branch: https://github.com/magma/magma/pull/8214
